src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPLINPACK.json

```diff
@@ -43,7 +43,7 @@
       "Parameters": {
         "Scenario": "InstallRequiredLinuxPackagesForHPL",
         "Repositories-Apt": "ppa:ubuntu-toolchain-r/test",
-        "Packages": "build-essential,linux-generic,libmpich-dev,libopenmpi-dev,libatlas-base-dev"
+        "Packages": "build-essential,libmpich-dev,libopenmpi-dev,libatlas-base-dev"
```

Comment:
That's a lot of packages, does this only work in ubuntu or apt store? How about other distro?

Reply:
Adding packages for other distros too.